src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,32 @@
-export const TodoInfo = () => {};
+import classNames from 'classnames';
+import React from 'react';
+import usersFromServer from '../../api/users';
+import { UserInfo } from '../UserInfo';
+
+type Todo = {
+  id: number;
+  title: string;
+  completed: boolean;
+  userId: number;
+};
+
+interface Props {
+  todo: Todo;
+}
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const userFind = usersFromServer.find(person => person.id === todo.userId);
```

Comment:
Consider renaming `userFind` to `foundUser` for better readability and consistency with naming conventions.